controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -2144,6 +2197,89 @@ int cli_bml::get_dcs_scan_results(const std::string &radio_mac, uint32_t max_res
     return 0;
 }
 
+/**
+ * get all client list.
+ * 
+ * @return 0 on success.
+ */
+int cli_bml::client_get_client_list()
+{
+    std::cout << __func__ << std::endl;
```

Comment:
I guess you added all of these `cout << __func__` print for debug, because these prints are not something we would like to see on the cli output. Please remove them all.